Assets/Scripts/Entity/Player/SimpleHeartDrop.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;

```

Comment:
Just want to note, that ideally we would have extracted the logic from the enemydrop script (Or exposed it somehow), so that it could be used here. Instead of duplicating code and now having to maintain two identical pieces of code, the  live in different places. Part of keeping your code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). Though, it doesnt matter at this point since we probably won't touch this code anymore.

Reply:
I did think of this actually. But the enemy drop script seemed to depend heavily on multiple variables and has an argument that is not even used lol.